app/models/concerns/project_media_getters.rb

```diff
@@ -174,7 +174,7 @@ def media_slug
   end
 
   def get_description
-    return self.fact_check_summary if self.get_main_channel == CheckChannels::ChannelCodes::FETCH
+    return self.fact_check_summary(true) if self.get_main_channel == CheckChannels::ChannelCodes::FETCH
```

Comment:
I think for `self.fact_check_summary(true)` we should not call cached field with true to enforce regenerate cached field and make sure that cached field updated in relevant actions

Reply:
There was a race condition so I had to force that refresh... the event was triggering a call to this method, so it would fail.